### PR TITLE
66 gdi frontend develop applications page

### PR DESCRIPTION
**Pull request by sulejmank**

### CODE REVIEW

File: src/types/application.types.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface ListedApplication {
```
Comment (inderps) [inline src/types/application.types.ts:1]: Any specific reason to call it `ListedApplication` and not simply `Application` or `DatasetsApplication`?

Comment (sulejmank) [reply src/types/application.types.ts:1]: I actually named it Application, but renamed it to match backend. We will anyway have also Application with more details

Comment (sulejmank) [reply src/types/application.types.ts:1]: But I am open for ideas :D

Comment (brunopacheco1) [reply src/types/application.types.ts:5]: I named it listed application as it's an abstraction of application, it contains only a few selected attributes. There will another call to fetch the complete application.

I aggree it should match the backend anyway.

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**